wpilibc/athena/include/DriverStation.h

```diff
@@ -99,30 +98,41 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   void Run();
   void UpdateControlWord(bool force, HAL_ControlWord& controlWord) const;
 
+  // Joystick User Data
   std::unique_ptr<HAL_JoystickAxes[]> m_joystickAxes;
   std::unique_ptr<HAL_JoystickPOVs[]> m_joystickPOVs;
   std::unique_ptr<HAL_JoystickButtons[]> m_joystickButtons;
   std::unique_ptr<HAL_JoystickDescriptor[]> m_joystickDescriptor;
 
-  // Cached Data
+  // Joystick Cached Data
   std::unique_ptr<HAL_JoystickAxes[]> m_joystickAxesCache;
   std::unique_ptr<HAL_JoystickPOVs[]> m_joystickPOVsCache;
   std::unique_ptr<HAL_JoystickButtons[]> m_joystickButtonsCache;
   std::unique_ptr<HAL_JoystickDescriptor[]> m_joystickDescriptorCache;
 
+  // Internal Driver Station thread
   Task m_task;
   std::atomic<bool> m_isRunning{false};
-  mutable Semaphore m_newControlData{Semaphore::kEmpty};
-  bool m_updatedControlLoopData = false;
+
+  // WPILib WaitForData control variables
+  bool m_waitForDataLoopCheck = false;
```

Comment:
The name of this variable doesn't tell me what it's for without context. I prefer naming them after boolean predicates. Maybe m_unblockWaitForData?